### ISSUE TITLE
Feature Request: SQL Server

### ISSUE THREAD

**Issue by Tam2** [ToolJet/ToolJet#212]
Is adding SQL Server as a data source on the roadmap? Would be good to get support for it

**Comment by Navaneeth-pk** [comment]
Yes, the SQL server is on the roadmap but we haven't prioritized it yet. We will update this issue when we have more information. 

Can you please tell us what your use case is if that's okay? It would be very helpful while building the feature.

**Comment by Tam2** [comment]
Our application uses sql server as the main database

I'm testing the tool out for now using our REST API as a data source instead 

However, seeing this error when trying to perform a request

```
ERROR:{1 item
message:string"src property must be a valid json object"
}
```

**Comment by Navaneeth-pk** [comment]
The preview expects the response from the API endpoint to be JSON and doesn't have a fallback for text. I will create a PR today for this.
Thanks for reporting the bug.

**Comment by Navaneeth-pk** [comment]
> Our application uses sql server as the main database
> 
> I'm testing the tool out for now using our REST API as a data source instead
> 
> However, seeing this error when trying to perform a request
> 
> ```
> ERROR:{1 item
> message:string"src property must be a valid json object"
> }
> ```

Can you please paste a sample response from your server? it would help me verify the fix. Thanks.

**Comment by Tam2** [comment]
Our API is sending a JSON response, is there somewhere where I can send you a link to the API privately so you can see the response?

**Comment by Navaneeth-pk** [comment]
Yes, my email is navaneeth@tooljet.io. 

Also, we have a slack community, feel free to join :) 

https://join.slack.com/t/tooljet/shared_invite/zt-r2neyfcw-KD1COL6t2kgVTlTtAV5rtg

**Comment by Navaneeth-pk** [comment]
@Tam2 Thanks for your help. The issue happens when the query pane is maximised or minimised after entering the `method` and `url`. A mistake made while building the REST API query editor. 

Will merge the fix after verifying on staging. https://github.com/ToolJet/ToolJet/pull/215

**Comment by alex-jitbit** [comment]
+1 I'd really love that feature.

SQL Server is more widely used than Postgres or Mongo or Elastic https://db-engines.com/en/ranking

**Comment by Navaneeth-pk** [comment]
We are working on integrating the MS SQL server :) 
Will be released in a couple of days. 

@alex-jitbit @josh-hemphill @Tam2

**Comment by Navaneeth-pk** [comment]
@alex-jitbit @josh-hemphill @Tam2

SQL Server integration has been merged to `develop`. Will be released after testing on staging.

**Comment by Navaneeth-pk** [comment]
Released in `v0.5.8`. Includes support for Azure SQL databases 🚀

@alex-jitbit @josh-hemphill @Tam2
